lib/tasks/add_navigation_links.rake

```diff
@@ -0,0 +1,56 @@
+# rubocop:disable Metrics/BlockLength
+namespace :navigation_links do
+  desc "Update DEV's navigation_links"
+  task update: :environment do
+    protocol = ApplicationConfig["APP_PROTOCOL"].freeze
+    domain = Rails.application&.initialized? ? SiteConfig.app_domain : ApplicationConfig["APP_DOMAIN"]
+    base_url = "#{protocol}#{domain}".freeze
+
+    reading_icon = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
+    thumb_up_icon = File.read(Rails.root.join("app/assets/images/twemoji/thumb-up.svg")).freeze
+    smart_icon = File.read(Rails.root.join("app/assets/images/twemoji/smart.svg")).freeze
+    look_icon = File.read(Rails.root.join("app/assets/images/twemoji/look.svg")).freeze
+    contact_icon = File.read(Rails.root.join("app/assets/images/twemoji/contact.svg")).freeze
+
+    # we can probably remove these icons once they in the database
+    listing_icon = File.read(Rails.root.join("app/assets/images/twemoji/listing.svg")).freeze
+    mic_icon = File.read(Rails.root.join("app/assets/images/twemoji/mic.svg")).freeze
+    camera_icon = File.read(Rails.root.join("app/assets/images/twemoji/camera.svg")).freeze
+    tag_icon = File.read(Rails.root.join("app/assets/images/twemoji/tag.svg")).freeze
+    bulb_icon = File.read(Rails.root.join("app/assets/images/twemoji/bulb.svg")).freeze
+    shopping_icon = File.read(Rails.root.join("app/assets/images/twemoji/shopping.svg")).freeze
+    heart_icon = File.read(Rails.root.join("app/assets/images/twemoji/heart.svg")).freeze
+    rainbowdev = File.read(Rails.root.join("app/assets/images/rainbowdev.svg")).freeze
+
+    NavigationLink.where(url: "#{base_url}/readinglist").first_or_create(name: "Reading List", icon: reading_icon,
```

Comment:
** comment (non-blocking):** This is a very minor thing, but I find this very hard to read. I cases like this one I usually use this form:

```ruby
 NavigationLink.where(url: "#{base_url}/readinglist").first_or_create(
  name: "Reading List",
  icon: reading_icon,
  display_only_when_signed_in: true,
  position: 0
)
```

Yes, it uses up slightly more vertical space but that's generally not a problem and it's much easier to follow as it moves the attributes towards the beginning of the line.

Reply:
✅